### ISSUE TITLE
alt Attributes

### ISSUE THREAD

**Issue by jordanwillis** [GIAallen/prj-rev-bwfs-tea-cozy#5]
Don't forget to always include some alternate text (e.g. `alt` attributes) on images that are important to your page content.

https://github.com/GIAallen/prj-rev-bwfs-tea-cozy/blob/master/Tea%20Cozy/index.html#L44
https://github.com/GIAallen/prj-rev-bwfs-tea-cozy/blob/master/Tea%20Cozy/index.html#L52
etc...

This accessibility feature it what allows browsers, search engines, and screen readers to understand the context of the image since they cannot "view" the image directly.  Here is an example...

``` html
<img src="https://s3.amazonaws.com/codecademy-content/courses/freelance-1/unit-4/img-myrtle-ave.jpg" alt="Myrtle Avenue Tea">
```
